src/Request/RequestFactoryInterface.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Psr\Http\Factory;
+namespace Psr\Http\Factory\Request;
```

Comment:
When there was only one of each interface it seemed fine, but then separating them made it into a mess:

```
CallbackStreamFactoryInterface.php
GlobalServerRequestFactoryInterface.php
RequestFactoryInterface.php
ResourceStreamFactoryInterface.php
ResponseFactoryInterface.php
ServerRequestFactoryInterface.php
StreamFactoryInterface.php
StringStreamFactoryInterface.php
UploadedFileFactoryInterface.php
UriFactoryInterface.php
```

Streams are mixed in all over the place and I don't really like `StreamFactoryCallbackInterface` so having a separate directory for each seemed to make more sense.